models/beer.py

```diff
@@ -0,0 +1,16 @@
+class Beer:
+    def __init__(self, name, link, explanation):
+        self.name = name
+        self.link = link
+        self.explanation = explanation
+
+    def __repr__(self):
+        return f"{self.name} - {self.link} - {self.explanation}"
+
+    def __hash__(self):
+        return hash(self.name)
+
+    def __eq__(self, other):
```

Comment:
Jak rozumiem nadpisujemy __eq__ i __hash__ (jak rozumiem jak w C# trzeba je razem nadpisywać) by on ogarnal te konwersję set(beers), by wykryć ze duplikaty to piwa z tymi samymi nazwami?

Reply:
tak dokładnie. Aktualnie ten `set(beers)` jest wykorzystywany tylko po to zeby wypisać na konsole liczbe znalezionych piw ogólnie, więc nawet nie wiem czy to będzie docelowo potrzebne ale na razie niech będzie